src/configManager.js

```diff
@@ -0,0 +1,46 @@
+const utils = require * "./utils.js";
+const fs = require("fs");
+
+let config;
+
+const defaultConfig = {
+  prefix: "$",
+  activity: { type: "playing", description: "$help" },
+  color: "purple",
+  plugins: {},
+  discordLogging: { active: true, level: "warn" },
+};
+
+function initConfig() {
+  fs.writeFile("../config.json", JSON.stringify(defaultConfig), function (err) {
+    if (err) throw "Error auto-generating configuration file.";
+    utils.logger.log(
```

Comment:
This is not run when creating a default config. Move outside the error callback for the `writeFile` call so it always runs when `initConfig` runs